config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -1,3 +1,67 @@
+periodics:
+- name: ci-azuredisk-csi-driver-e2e-capz-windows
+  cluster: eks-prow-build-cluster
+  interval: 72h
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-community: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: release-1.17
```

Comment:
this will need to be `main` to run against k/k main branch since we need: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5443

Reply:
good catch

Reply:
I'm working on cherry-picking this change to 1.17 and 1.18